### PR TITLE
fix(flow-288): fix for double pricing table rendering

### PR DESCRIPTION
**Pull request by adaptajoe**
As per the title. I'd recommend opening a Hello World app to test this out. I did so and it _seemed_ to be fine, but best to double-check.

### CODE REVIEW

File: packages/pricing-table/src/components/SalablePricingTableReact/SalablePricingTableReact.tsx [hunk 1]
```diff
@@ -25,7 +25,9 @@ export const SalablePricingTableReact: FC<IPricingTableReact> = ({ envConfig, ch
         if (window.SalablePricingTable) {
           (async () => {
             pricingTable = new window.SalablePricingTable(envConfig, checkoutConfig);
-            await pricingTable?.init();
+            document.querySelector('.salable-pricing-table-container')
```
Comment (rahuladaptavist) [inline packages/pricing-table/src/components/SalablePricingTableReact/SalablePricingTableReact.tsx:28]: Can we put this inside if and then initialise the pricing table

**Review by conermurphy — CHANGES_REQUESTED** [review]
Just one comment. Also once this is merged, don't forget to make a PR into the main repo to bump the version of the `react-sdk`.

File: packages/pricing-table/src/components/SalablePricingTableReact/SalablePricingTableReact.tsx (hunk 1 above)
Comment (adaptajoe) [reply packages/pricing-table/src/components/SalablePricingTableReact/SalablePricingTableReact.tsx]: Fixed in upcoming commit

**Comment by adaptajoe** [comment]
@conermurphy @rahuladaptavist @avst-perrygeorge please re-review so we can get this out! 😄

**Review by conermurphy — APPROVED** [review]

**Comment by conermurphy** [comment]
After this is merged in and released, we'll need to make a PR to salable-mono to bump the package version in the `package.json`

**Comment by adaptajoe** [comment]
@rahuladaptavist please review.

**Review by adapta-sean — APPROVED** [review]

**Pull request merged**